### PR TITLE
[MI-2976] Auto fill the caller field if the sender is connected to Se…

### PR DESCRIPTION
**Pull request by ayusht2810**
…rviceNow

### CODE REVIEW

**Review by Kshitij-Katiyar — APPROVED** [review]

File: webapp/src/containers/createIncident/callerPanel.tsx [hunk 1]
```diff
@@ -57,6 +59,15 @@ const CallerPanel = (({
         handleSuccess: () => {
             setApiError(null);
             setOptions(data);
+            if (senderId) {
+                const senderDetails = data?.filter((c) => c.mattermostUserID === senderId)[0];
```
Comment (raghavaggarwal2308) [inline webapp/src/containers/createIncident/callerPanel.tsx:63]: Do we need to check the length here?

**Review by raghavaggarwal2308 — CHANGES_REQUESTED** [review]

File: webapp/src/containers/createIncident/callerPanel.tsx (hunk 1 above)
Comment (raghavaggarwal2308) [reply webapp/src/containers/createIncident/callerPanel.tsx:63]: discussed

Comment (ayusht2810) [reply webapp/src/containers/createIncident/callerPanel.tsx:63]: discussed

**Review by raghavaggarwal2308 — APPROVED** [review]

File: webapp/src/containers/createIncident/callerPanel.tsx (hunk 1 above)
Comment (manojmalik20) [inline webapp/src/containers/createIncident/callerPanel.tsx:63]: Use `find` instead of `filter`.

**Review by manojmalik20 — APPROVED** [review]

**Review by manojmalik20 — APPROVED** [review]

**Pull request merged**